app/styleguides/_default/components/tweet.css

```diff
@@ -0,0 +1,48 @@
+.clay-tweet {
+    display: block;
+    margin: 0 0 24px;
+    width: 100%;
+
+    & blockquote.twitter-tweet {
+      border-color: #eee #ddd #bbb;
```

Comment:
I don't know if you noticed in previous examples, but we have a convention to store hex colors and font stacks in variables to be able to reuse them. Please do this for every value of color and font in this file. [Here's an example](https://github.com/nymag/sites/blob/master/styleguides/developing/components/author.css)